packages/backend/src/database/entities/jobs.ts

```diff
@@ -10,14 +10,28 @@ export type DbJobs = {
     created_at: Date;
     updated_at: Date;
     job_status: JobStatusType;
+    job_type: Job['jobType'];
+    results?: Job['jobResults'];
```

Comment:
database entity typing is a bit lazy, doesn't enforce that specific `job_type` values have specfic `results`